packages/components/src/components/file-dropzone/file-dropzone.tsx

```diff
@@ -57,8 +88,9 @@ const PreviewSingle = props => {
     );
 };
 
-const FileDropzone = ({ className, noClick = false, ...props }) => {
+const FileDropzone = ({ className, noClick = false, ...props }: TFileDropzone) => {
```

Comment:
The idea of spreading the props `..props` can reduce code readability.
is it possible to remove it and add needed ones one by one?

Reply:
Since the props are further passed down to Preview component, I think it is better this way.